sqlservice/client.py

```diff
@@ -119,8 +123,12 @@ def __init__(self,
             'SQL_MAX_OVERFLOW': None,
             'SQL_AUTOCOMMIT': False,
             'SQL_AUTOFLUSH': True,
-            'SQL_EXPIRE_ON_COMMIT': True
+            'SQL_EXPIRE_ON_COMMIT': True,
         }
+        
+        # pool_pre_ping was added in SQLAlchemy 1.2.0
+        if sa.__version__ >= '1.2.0':
+            self.config['SQL_POOL_PRE_PING'] = True
```

Comment:
I'm fine with adding `SQL_POOL_PRE_PING` as a config option but it shouldn't default to ``True``. Instead it should be opt-in since it's not always desirable to have it enabled.